core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -535,6 +535,34 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey WEB_CORS_ALLOW_CREDENTIAL =
+      new Builder(Name.WEB_CORS_ALLOW_CREDENTIAL)
+          .setDefaultValue("false")
+          .setDescription("Enable request include credential.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.SERVER)
+          .build();
+  public static final PropertyKey WEB_CORS_ALLOW_HEADERS =
+      new Builder(Name.WEB_CORS_ALLOW_HEADERS)
+          .setDefaultValue("*")
```

Comment:
should default be `GET, POST, OPTIONS` (what was defined in `RestUtils`)?

Reply:
I don't know why the methods were restricted to GET, POST, OPTIONS before. i think as a default, `*` is a better choice.